satellite/satellitedb/accounting.go

```diff
@@ -225,12 +227,23 @@ func (db *accountingDB) SaveBucketTallies(ctx context.Context, intervalStart tim
 		iSegments := dbx.BucketStorageTally_InlineSegmentsCount(uint(info.InlineSegments))
 		objectCount := dbx.BucketStorageTally_ObjectCount(uint(info.Files))
 		meta := dbx.BucketStorageTally_MetadataSize(uint64(info.MetadataSize))
-		_, err := db.db.Create_BucketStorageTally(ctx, bucketName, projectID, interval, inlineBytes, remoteBytes, rSegments, iSegments, objectCount, meta)
+		dbxTally, err := db.db.Create_BucketStorageTally(ctx, bucketName, projectID, interval, inlineBytes, remoteBytes, rSegments, iSegments, objectCount, meta)
 		if err != nil {
-			return err
+			return nil, err
 		}
+		tally := accounting.BucketTally{
```

Comment:
Rather than posting to the DB and creating a slice of BucketTallies in one method, do you think we should split them into 2 methods, one for each? That would avoid returning the slice and then dropping it.

Reply:
yeah that sounds like a good idea.